src-java/floodlightrouter-topology/floodlightrouter-storm-topology/src/main/java/org/openkilda/wfm/topology/floodlightrouter/bolts/SwitchMonitorBolt.java

```diff
@@ -101,7 +106,24 @@ public void regionUpdateNotification(RegionMappingUpdate mappingUpdate) {
     }
 
     @Override
-    public void networkStatusUpdateNotification(SwitchId switchId, InfoData notification) {
+    public void sendSwitchConnectNotification(
+            SwitchId switchId, SpeakerSwitchView speakerData, SwitchAvailabilityData availabilityData) {
+        sendOtherNotification(switchId, new SwitchConnectNotification(speakerData, availabilityData));
+    }
+
+    @Override
+    public void sendSwitchDisconnectNotification(
+            SwitchId switchId, SwitchAvailabilityData availabilityData, boolean isRegionOffline) {
+        sendOtherNotification(switchId, new SwitchDisconnectNotification(switchId, availabilityData, isRegionOffline));
+    }
+
+    @Override
+    public void sendSwitchAvailabilityUpdateNotification(SwitchId switchId, SwitchAvailabilityData availabilityData) {
+        sendOtherNotification(switchId, new SwitchAvailabilityUpdateNotification(switchId, availabilityData));
+    }
+
+    @Override
+    public void sendOtherNotification(SwitchId switchId, InfoData notification) {
```

Comment:
nit: other to what?

Reply:
"other" to what we know. 

SwitchMonitor handling in a special way some subset of events (known events are transferred with dedicated carrier's methods), other events (not known) just trasferred as is (other events).